### PR TITLE
[llm]fix shift label

### PR DESCRIPTION
**Pull request by lugimzzz**
### PR types
Bug fixes 

### PR changes
Others 

### Description
修复shift label问题

### CODE REVIEW

File: examples/language_model/chatglm/utils.py [hunk 1]
```diff
@@ -58,14 +58,20 @@ def prediction_step(
...
+                return (loss, logits[0].argmax(axis=-1, keepdim=True), labels)
+
         loss = None
 
         n_token_id = self.tokenizer.convert_tokens_to_ids("<n>")
         model.eval()
         with paddle.no_grad():
             generated_tokens = model.generate(
-                **inputs,
+                input_ids=inputs["input_ids"],
```
Comment (lugimzzz) [inline examples/language_model/chatglm/utils.py:72]: 传入label的话，模型forward会去计算loss导致出错

File: examples/language_model/bloom/finetune_generation.py [hunk 2]
```diff
@@ -235,17 +238,17 @@ def main():
         if model_args.qat_type == "A8W8":
             activation = PACTQuanter(quanter=FakeQuanterWithAbsMaxObserverLayer, init_value=20, dtype=dtype)
             # activation = FakeQuanterWithAbsMaxObserver(moving_rate=0.9, bit_length=8, dtype=dtype)
-            # weight = FakeQuanterChannelWiseAbsMaxObserver(bit_length=8, dtype=dtype)
-            weight = FakeQuanterWithAbsMaxObserver(bit_length=8, dtype=dtype)
+            weight = FakeQuanterChannelWiseAbsMaxObserver(bit_length=8, dtype="float32")
```
Comment (lugimzzz) [inline examples/language_model/bloom/finetune_generation.py:241]: channel wise 只支持float32

**Review by sijunhe — APPROVED** [review]
lgtm
